src/api_routes_http.rs

```diff
@@ -340,6 +341,7 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/delete", web::post().to(delete_custom_emoji)),
       ),
   );
+  cfg.service(web::scope("/sitemap.xml").route("", web::get().to(get_sitemap)));
```

Comment:
Chaining this seems to always affect the formatting of the whole routing setup (since it's not scoped to "/api/v3/"). Open to suggestions how to handle this, but this at least makes my change more obvious

Reply:
Might be worth to wrap this with rate_limit.message() as well. Otherwise the service is unnecessary and you can use the route directly.